src/SDKs/Authorization/Management.Authorization/Microsoft.Azure.Management.Authorization.csproj

```diff
@@ -13,11 +13,12 @@
 		<PackageReleaseNotes>
```

Comment:
There are some breaking changes being introduced in the SDK, the nuget version must be bumped